.github/utils/tutorials.sh

```diff
@@ -61,15 +62,50 @@ for script in $scripts_to_run; do
         echo "NOT using reduced GoT dataset!"
     fi
 
+    # FIXME Make the Python path editable (thanks espnet... -.-")
+    if [[ $make_python_path_editable == "EDITABLE" ]] && [[ "$script" == *"Tutorial17_"* ]]; then
+        sudo find $python_path/lib -type f -exec chmod 777 {} \;
+    fi
+
     if [[ "$script" == *".py" ]]; then
         time python $script
     else
         sudo $python_path/bin/ipython -c "%run $script"
     fi
+
+    if [ ! $? -eq 0 ]; then
+        failed=$failed" "$script
+    fi
+
+    # Clean up datasets and SQLite DBs to avoid crashing the next tutorial
     git clean -f
 
 done
 
 # causes permission errors on Post Cache
 sudo rm -rf data/
-sudo rm -rf /home/runner/work/haystack/haystack/elasticsearch-7.9.2/
\ No newline at end of file
+sudo rm -rf /home/runner/work/haystack/haystack/elasticsearch-7.9.2/
+
+
+if [[ $failed == "" ]]; then
+    echo ""
+    echo "##################################################################################"
+    echo "##                                                                              ##"
+    echo "##                   All tutorials executed successfully                        ##"
+    echo "##                                                                              ##"
+    echo "##################################################################################"
+    exit 0
+
+else
+    echo ""
+    echo "##################################################################################"
+    echo "##                                                                              ##"
+    echo "##                        Some tutorials have failed!                           ##"
+    echo "##                                                                              ##"
+    echo "##################################################################################"
+    for script in $failed; do
+    echo "##  - $script"
+    done
+    echo "##################################################################################"
+    exit 1
+fi
```

Comment:
I've realized is better if all tutorials run, regardless of the status of previous tutorials. I now made all tutorials run in all cases, and make a recap of which ones failed at the end of the log.